README.md

```diff
@@ -150,17 +150,19 @@ You can also create [issue](https://github.com/privacy-scaling-explorations/snar
 
 ### Partial clone
 
-For a more manageable clone that includes only the packages relevant to you or none of them, you can use git's [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) and [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt) features. This will reduce clone time and improve git performance.
+For a more manageable clone that includes only the packages relevant to you or none of them, we use git's [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) and [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt) features. This will reduce clone time and improve git performance.
 
-```bash
-git clone --sparse --filter=blob:none <forkedUrl>
+For convenience we provide an interactive [`setup`](https://github.com/privacy-scaling-explorations/snark-artifacts/tree/main/scripts/bin/setup) script.  
+It will ask you where to clone the repository, and what packages you want to sparse checkout.
+
+``` bash
+bash <(curl https://raw.githubusercontent.com/privacy-scaling-explorations/snark-artifacts/main/scripts/bin/setup) fetch
```

Comment:
shouldn't there be `fetch setup` at the end?

Reply:
the second positional arg is to override the branch where the script is located and fetched from.
When this PR will have merged, this will always be `main`.
While unmerged and for testing, I wanted a way to override it and indicate the script are located on the `setup` branch.

https://github.com/privacy-scaling-explorations/snark-artifacts/blob/23643abd8be669cb2d6658b85bf66c4ddbc0a8ef/scripts/bin/setup#L47-L48

Reply:
I see! Thanks